.github/workflows/pr-automation.yml

```
@@ -0,0 +1,250 @@
+          scopes: |
+            backend
+            frontend
+            config
+            ci
+          requireScope: false
+          wip: true
+        continue-on-error: true
+
+      - name: Debug semantic PR outputs
+        if: always()
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)